src/lsm/bloom_filter.zig

```diff
@@ -116,8 +117,9 @@ const test_bloom_filter = struct {
 
 test "bloom filter: random" {
     var rng = std.rand.DefaultPrng.init(42);
-    var i: usize = 0;
-    while (i < (1 << 12)) : (i += 1) {
-        try test_bloom_filter.random_keys(rng.random(), i);
+    const max_iters: usize = (1 << 12);
+    var iter: usize = 0;
```

Comment:
Let's do `iteration` and `iterations_max` (following TigerStyle, variable names in full, and variable name components ordered by MSB—most significant word first).

Or just `index` and `count`.